django/core/files/storage.py

```diff
@@ -140,24 +149,102 @@ def url(self, name):
+        specified by name. Deprecated: use get_created_time() instead.
         """
+        warnings.warn(
+            'Storage.created_time() is deprecated. '
+            'Use Storage.get_created_time() instead.',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
         raise NotImplementedError('subclasses of Storage must provide a created_time() method')
 
     def modified_time(self, name):
         """
         Returns the last modified time (as datetime object) of the file
-        specified by name.
+        specified by name. Deprecated: use get_modified_time() instead.
         """
+        warnings.warn(
+            'Storage.modified_time() is deprecated. '
+            'Use Storage.get_modified_time() instead.',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
         raise NotImplementedError('subclasses of Storage must provide a modified_time() method')
 
+    def get_accessed_time(self, name):
+        """
+        Return the last accessed time (as a datetime object) of the file
+        specified by name. The datetime object will be timezone-aware if
+        USE_TZ=True.
+        """
+        warnings.warn(
+            'Storage.accessed_time() is deprecated. '
+            'Storage backends should implement get_accessed_time().',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
+        dt = self.accessed_time(name)
+        return _possibly_make_aware(dt)
+
+    def get_created_time(self, name):
+        """
+        Return the creation time (as a datetime object) of the file
+        specified by name. The datetime object will be timezone-aware if
+        USE_TZ=True.
+        """
+        warnings.warn(
+            'Storage.created_time() is deprecated. '
+            'Storage backends should implement get_created_time().',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
+        dt = self.created_time(name)
+        return _possibly_make_aware(dt)
+
+    def get_modified_time(self, name):
+        """
+        Return the last modified time (as a datetime object) of the file
+        specified by name. The datetime object will be timezone-aware if
+        USE_TZ=True.
+        """
+        warnings.warn(
+            'Storage.modified_time() is deprecated. '
+            'Storage backends should implement get_modified_time().',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
+        dt = self.modified_time(name)
+        return _possibly_make_aware(dt)
+
+
+def _possibly_make_aware(dt):
+    """
+    Convert a datetime object in the local timezone to aware
+    if USE_TZ is True.
+    """
+    if settings.USE_TZ:
+        tzname = os.environ['TZ']
+        if isinstance(tzname, six.string_types) and pytz is not None:
+            tz = pytz.timezone(tzname)
+        else:
+            tz = timezone.LocalTimezone()
+        dt = timezone.make_aware(dt, tz)
```

Comment:
I believe you can replace this whole block with:

    tz = timezone.get_current_timezone()
    dt = timezone.make_aware(dt, tz)